### PR TITLE
private/pedro/font missing to jsdialog T39589 T39329 T39075

### PR DESCRIPTION
**Pull request by pedropintosilva**
- Missing Fonts dialog: Use JSDialog instead of Vex, allow dismiss
- Fix Missing title on showInfoModal when on a mobile

### CODE REVIEW

File: browser/src/core/Socket.js [hunk 1]
```diff
@@ -1077,14 +1077,14 @@ app.definitions.Socket = L.Class.extend({
...
 			var fontsMissingObj = JSON.parse(textMsg.substring(textMsg.indexOf('{')));
-			var msg = _('Missing fonts:');
-			msg += ' ';
+			var msg = ' ';
 			for (var i = 0; i < fontsMissingObj.fontsmissing.length; ++i) {
 				if (i > 0)
 					msg += ', ';
 				msg += fontsMissingObj.fontsmissing[i];
 			}
 			this._map.fire('infobar', { msg: msg });
```
Comment (eszkadev) [inline browser/src/core/Socket.js:1086]: we should not use infobar anymore after it is converted, right?

File: browser/src/core/Socket.js [hunk 2]
```diff
@@ -1077,14 +1077,14 @@ app.definitions.Socket = L.Class.extend({
...
+			this._map.uiManager.showInfoModal('fontsmissing', _('Missing Fonts'), msg, null, _('Close'));
```
[the remark this answers is not in the document]
Comment (pedropintosilva) [reply browser/src/core/Socket.js]: don't worry about that, we can then make sure that every showinfomodal dialog has minimum width and max width

[the remark this answers is not in the document]
Comment (pedropintosilva) [reply browser/src/core/Socket.js]: @gulsah let's add in the css (jsdialog.css)
```css
min-width: 198px;
max-width: 600px;
```

**Review by eszkadev — COMMENTED** [review]
Looks good now (but I don't know how to trigger that so haven't tested)

**Comment by GulsahKose** [comment]
Szymon to see a missing font warning you need to have a document with a font that not installed in you system. for a test I create a document with "gargi" font. Then i deleted that font from my operating system.

Then In you coolwsd.xml following setting:

    <fonts_missing>
        <handling desc="How to handle fonts mising in a document: 'report', 'log', 'both', or 'ignore'" type="string" default="log">both</handling>
    </fonts_missing>

To see welcome dialog you need to clear your browser cache. Then you can test.

**Comment by pedropintosilva** [comment]
this was already approved before

**Comment by pedropintosilva** [comment]
and I just , once more, rebased it

**Pull request merged**